File: frontend-react/fix_imports.py
import os
import re

def walk_and_replace(directory):
    for root, dirs, files in os.walk(directory):
        for file in files:
            if file.endswith('.tsx') or file.endswith('.ts'):
                file_path = os.path.join(root, file)
                with open(file_path, 'r', encoding='utf-8') as f:
                    content = f.read()
                
                original = content
                content = re.sub(r'from "(\.\.\/\.\.\/services)\/(admin[a-zA-Z0-9_]+)', r'from "\1/admin/\2', content)
                content = re.sub(r"from '(\.\.\/\.\.\/services)\/(admin[a-zA-Z0-9_]+)", r"from '\1/admin/\2", content)
                content = re.sub(r'from "(\.\.\/\.\.\/\.\.\/services)\/(admin[a-zA-Z0-9_]+)', r'from "\1/admin/\2', content)
                content = re.sub(r"from '(\.\.\/\.\.\/\.\.\/services)\/(admin[a-zA-Z0-9_]+)", r"from '\1/admin/\2", content)
                
                if content != original:
                    with open(file_path, 'w', encoding='utf-8') as f:
                        f.write(content)
                    print(f"Updated {file_path}")

walk_and_replace('./src')

File: frontend-react/fix_imports.js
const fs = require('fs');
const path = require('path');

function walkDir(dir, callback) {
  fs.readdirSync(dir).forEach(f => {
    let dirPath = path.join(dir, f);
    let isDirectory = fs.statSync(dirPath).isDirectory();
    isDirectory ? walkDir(dirPath, callback) : callback(path.join(dir, f));
  });
}

walkDir('./src', (filePath) => {
  if (filePath.endsWith('.tsx') || filePath.endsWith('.ts')) {
    let content = fs.readFileSync(filePath, 'utf8');
    let original = content;
    
    // Replace ../../services/adminX with ../../services/admin/adminX
    content = content.replace(/from "(\.\.\/\.\.\/services)\/(admin[a-zA-Z0-9_]+)/g, 'from "$1/admin/$2');
    content = content.replace(/from '(\.\.\/\.\.\/services)\/(admin[a-zA-Z0-9_]+)/g, "from '$1/admin/$2");
    
    // Replace ../../../services/adminX with ../../../services/admin/adminX
    content = content.replace(/from "(\.\.\/\.\.\/\.\.\/services)\/(admin[a-zA-Z0-9_]+)/g, 'from "$1/admin/$2');
    content = content.replace(/from '(\.\.\/\.\.\/\.\.\/services)\/(admin[a-zA-Z0-9_]+)/g, "from '$1/admin/$2");
    
    if (content !== original) {
      fs.writeFileSync(filePath, content, 'utf8');
      console.log(`Updated ${filePath}`);
    }
  }
});
